src/testFixtures/java/uk/gov/di/ipv/cri/common/library/aws/CloudFormationHelper.java

```diff
@@ -1,81 +1,59 @@
 package uk.gov.di.ipv.cri.common.library.aws;
 
 import software.amazon.awssdk.services.cloudformation.CloudFormationClient;
-import software.amazon.awssdk.services.cloudformation.model.DescribeStacksRequest;
-import software.amazon.awssdk.services.cloudformation.model.DescribeStacksResponse;
+import software.amazon.awssdk.services.cloudformation.model.Output;
+import software.amazon.awssdk.services.cloudformation.model.Parameter;
 import software.amazon.awssdk.services.cloudformation.model.Stack;
 
 import java.security.InvalidParameterException;
-import java.util.List;
 
 /**
- * A utility class to retrieve stack and it's outputs and parameters. The class is a wrapper around
- * {@link CloudFormationClient} and allows to get around retrieving the mock SQS url used for tests
+ * A wrapper class around {@link CloudFormationClient} to retrieve information about CloudFormation
+ * stacks
  */
 public final class CloudFormationHelper {
 
-    /**
-     * Client for accessing AWS CloudFormation. All service calls made using this client are
-     * blocking, and will not return until the service call completes.
-     */
-    private static final CloudFormationClient cloudFormation = CloudFormationClient.create();
+    private static final CloudFormationClient CLOUD_FORMATION = CloudFormationClient.create();
 
     private CloudFormationHelper() {}
 
-    /**
-     * Returns the parameter data type from the stack. ParameterKey - The key associated with the
-     * parameter. If you don't specify a key and value for a particular parameter, AWS
-     * CloudFormation uses the default value that's specified in your template. ParameterValue - The
-     * input value associated with the parameter.
-     */
+    /** Get a parameter value from a stack */
     public static String getParameter(String stackName, String parameterName) {
         return getStack(stackName).parameters().stream()
                 .filter(parameter -> parameter.parameterKey().equals(parameterName))
+                .map(Parameter::parameterValue)
                 .findFirst()
                 .orElseThrow(
                         () ->
                                 new InvalidParameterException(
```

Comment:
Please can we revert this back to using IllegalArgumentException as we should not be using java.security exceptions. Thanks

Reply:
Good catch! Done ✅